bokeh/core/properties.py

```diff
@@ -232,6 +233,7 @@ class SomeModel(Model):
     'RGB',
     'Regex',
     'RelativeDelta',
+    'RestrictedDict',
```

Comment:
Is this not required? (adding `RestrictedDict` here throws an error, removing it doesn't)